forloop_modules/function_handlers/variable_handlers.py

```diff
@@ -16,6 +16,7 @@
 
 from forloop_modules.function_handlers.auxilliary.abstract_function_handler import AbstractFunctionHandler, Input
 from forloop_modules.errors.errors import CriticalPipelineError
+from forloop_modules.function_handlers.input_waiter import register_pending_input
```

Comment:
function_handlers/auxilliary/input_waiter -> in function handlers folder there shouldnt be different types than function handlers